test/test_web.py

```diff
@@ -218,38 +218,38 @@ def mc_server(self, use_haproxy: bool) -> mock.MagicMock:
         )
         mc_server.sensors.add(
             Sensor(
-                str,
+                bytes,
                 "gui-urls",
                 "",
-                default=json.dumps(ROOT_GUI_URLS),
+                default=json.dumps(ROOT_GUI_URLS).encode(),
                 initial_status=Sensor.Status.NOMINAL,
```

Comment:
I think this one should stay as `str`, because it's created directly in the master controller server (as `str`). The productN.gui-urls are created by SensorWatcher and so will have type `bytes`.